zipline-cli/build.gradle.kts

```diff
@@ -0,0 +1,73 @@
+import com.android.build.gradle.internal.tasks.factory.dependsOn
+import com.vanniktech.maven.publish.JavadocJar
+import com.vanniktech.maven.publish.KotlinJvm
+import org.apache.tools.ant.taskdefs.condition.Os
+
+plugins {
+  kotlin("jvm")
+  kotlin("kapt")
+  id("org.jetbrains.dokka")
+  id("com.vanniktech.maven.publish.base")
+  id("com.palantir.graal")
+  id("com.github.johnrengelman.shadow")
+}
+
+tasks.jar {
+  manifest {
+    attributes("Automatic-Module-Name" to "app.cash.zipline.cli")
+    attributes("Main-Class" to "app.cash.zipline.cli.Main")
+  }
+}
+
+// resources-templates.
+sourceSets {
+  main {
+    resources.srcDirs("$buildDir/generated/resources-templates")
+  }
+}
+
+dependencies {
+  api(project(":zipline-loader"))
+  implementation(Dependencies.okHttp)
+  implementation(Dependencies.picocli)
+
+  kapt(Dependencies.picocliCompiler)
+
+  testImplementation(Dependencies.junit)
+  testImplementation(Dependencies.kotlinTest)
+}
+
+tasks.shadowJar {
+  mergeServiceFiles()
+}
+
+graal {
```

Comment:
We would need to build this on all three platforms on CI, right? Otherwise you have to build it locally yourself first to get the correct version for your host OS. Should we just use the application plugin which produces a zip containing a version that runs on all three platforms and starter scripts? I don't think there's any advantage to graal except a lot of extra build configuration.